src/ripple/app/tx/impl/NFTokenModify.cpp

```diff
@@ -0,0 +1,96 @@
+
+#include <ripple/app/tx/impl/NFTokenModify.h>
```

Comment:
This new transaction type needs to be added to `ripple/protocol/TxFormats.h` and `ripple/protocol/impl/TxFormats.cpp` and `ripple/app/tx/impl/applySteps.cpp` - otherwise rippled doesn't know it exists